### ISSUE TITLE
telegraf READ.ME - Minimal example to start an InfluxDB container fails with the current versions

### ISSUE THREAD

**Issue by k-byun** [influxdata/influxdata-docker#685]
As per the [telegraf Read.me](https://hub.docker.com/_/telegraf) for containerized instance setup, for the minimal setup:

```bash
docker run -d --name influxdb -p 8086:8086 influxdb
docker run --net=container:influxdb telegraf
```

generates the output:

```
2023-05-19T14:52:11Z I! Loading config: /etc/telegraf/telegraf.conf
2023-05-19T14:52:11Z E! [telegraf] Error running agent: no outputs found, did you provide a valid config file?
```

I ran `docker run --rm -it --name telegraf-temp1 telegraf bash` to see what the `telegraf.conf` file has, and found out the output sections are all commented out. Should the ReadMe.md be updated because the two commands suggested to run with the minimal setup does not work, and is mis-leading.

**Comment by powersj** [comment]
agreed, let me get a PR for that. I believe https://github.com/docker-library/docs/tree/master/telegraf is the correct repo/file.

**Comment by powersj** [comment]
PR up https://github.com/docker-library/docs/pull/2328

**Comment by k-byun** [comment]
I see `README.md ` has been updated. Closing the issue.
